go.mod

```diff
@@ -3,67 +3,21 @@ module github.com/cosmos/sdk-application-tutorial
-	github.com/cosmos/cosmos-sdk v0.28.2-0.20190604125914-75de63ce3166
-	github.com/cosmos/go-bip39 v0.0.0-20180618194314-52158e4697b8
-	github.com/cosmos/ledger-cosmos-go v0.10.3
-	github.com/davecgh/go-spew v1.1.1
-	github.com/ethereum/go-ethereum v1.8.23
-	github.com/fsnotify/fsnotify v1.4.7
-	github.com/go-kit/kit v0.8.0
-	github.com/go-logfmt/logfmt v0.4.0
-	github.com/go-stack/stack v1.8.0
-	github.com/gogo/protobuf v1.1.1
-	github.com/golang/protobuf v1.3.0
-	github.com/golang/snappy v0.0.1
+	github.com/cosmos/cosmos-sdk v0.35.0
 	github.com/gorilla/mux v1.7.0
-	github.com/gorilla/websocket v1.4.0
-	github.com/hashicorp/hcl v1.0.0
-	github.com/inconshreveable/mousetrap v1.0.0
-	github.com/jmhodges/levigo v1.0.0
-	github.com/kr/logfmt v0.0.0-20140226030751-b84e30acd515
-	github.com/magiconair/properties v1.8.0
-	github.com/mattn/go-isatty v0.0.7
-	github.com/matttproud/golang_protobuf_extensions v1.0.1
-	github.com/mitchellh/mapstructure v1.1.2
-	github.com/otiai10/copy v0.0.0-20180813032824-7e9a647135a1 // indirect
-	github.com/otiai10/curr v0.0.0-20150429015615-9b4961190c95 // indirect
-	github.com/otiai10/mint v1.2.3 // indirect
-	github.com/pelletier/go-toml v1.2.0
-	github.com/pkg/errors v0.8.1
-	github.com/pmezard/go-difflib v1.0.0
-	github.com/prometheus/client_golang v0.9.2
-	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
-	github.com/prometheus/common v0.2.0
-	github.com/prometheus/procfs v0.0.0-20190328153300-af7bedc223fb
-	github.com/rakyll/statik v0.1.4
-	github.com/rcrowley/go-metrics v0.0.0-20180503174638-e2704e165165
-	github.com/rs/cors v1.6.0
-	github.com/spf13/afero v1.2.2
-	github.com/spf13/cast v1.3.0
+	github.com/mattn/go-isatty v0.0.7 // indirect
+	github.com/prometheus/procfs v0.0.0-20190328153300-af7bedc223fb // indirect
+	github.com/spf13/afero v1.2.2 // indirect
 	github.com/spf13/cobra v0.0.3
-	github.com/spf13/jwalterweatherman v1.1.0
-	github.com/spf13/pflag v1.0.3
 	github.com/spf13/viper v1.0.3
-	github.com/stretchr/testify v1.3.0
-	github.com/syndtr/goleveldb v1.0.0
-	github.com/tendermint/btcd v0.1.1
+	github.com/syndtr/goleveldb v1.0.0 // indirect
 	github.com/tendermint/go-amino v0.15.0
-	github.com/tendermint/iavl v0.12.2
 	github.com/tendermint/tendermint v0.31.5
-	github.com/zondax/hid v0.9.0
-	github.com/zondax/ledger-go v0.8.0
-	golang.org/x/net v0.0.0-20190213061140-3a22650c66bd
-	golang.org/x/sys v0.0.0-20190329044733-9eb1bfa1ce65
-	golang.org/x/text v0.3.0
-	google.golang.org/genproto v0.0.0-20190327125643-d831d65fe17d
-	google.golang.org/grpc v1.19.1
-	gopkg.in/yaml.v2 v2.2.2
+	golang.org/x/sys v0.0.0-20190329044733-9eb1bfa1ce65 // indirect
+	google.golang.org/genproto v0.0.0-20190327125643-d831d65fe17d // indirect
+	google.golang.org/grpc v1.19.1 // indirect
 )
 
 replace golang.org/x/crypto => github.com/tendermint/crypto v0.0.0-20180820045704-3764759f34a5
+
+replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.28.2-0.20190605232616-5f9c3fdf8895
```

Comment:
will leave this till the next release

Reply:
I think we can just update the existing version instead of using a replace (and maybe add a TODO).